packages/text-input/text-input.scss

```diff
@@ -27,13 +27,31 @@ $textarea-height--expanded: $input-height * 7 + $bottom-padding;
     min-height: $input-total-height;
     padding-top: $label-height;
     background-color: $input-bg-color;
-    padding-bottom: $component-spacing--large;
 
     &--inline {
         display: inline-block;
         vertical-align: bottom;
     }
 
+    &__input:focus ~ .jkl-text-field__placeholder {
+        color: darkgray;
+        transform: translate3d(0, 0, 0);
+    }
+
+    &__placeholder {
+        @include font-size(small);
+        @include line-height(small);
+
+        color: transparent;
+        user-select: none;
+        left: 0;
+        padding-left: 0.125rem;
+        position: absolute;
+        transform: translate3d(0, 20px, 0);
```

Comment:
Why 3d? And should the px value be a variable?

Reply:
☝️